cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/etl/transform/ScriptTransform.java

```diff
@@ -93,9 +96,14 @@
     @Nullable
     private final String schema;
 
-    public Config(String script, String schema) {
+    @Description("Lookup tables to use during transform")
```

Comment:
Also, you need to edit the example at the top of this file and also examples in docs to include info about how you can do lookups..  same with other transforms..